lms/djangoapps/verify_student/models.py

```diff
@@ -1085,6 +1085,14 @@ def get_user_attempts(cls, user_id, course_key, related_assessment, location_id)
             location_id=location_id,
             status="submitted"
         ).count()
+    
+    @classmethod
+    def get_location_id(cls, photo_verification):
+        try:
+            ver_status = cls.objects.filter(checkpoint__photo_verification=photo_verification).latest()
```

Comment:
Make this one line:
`return cls.objects.filter(checkpoint__photo_verification=photo_verification).latest().location_id`